build/config/PackageSignToolData.json

```diff
@@ -0,0 +1,27 @@
+{
+    "sign": [
+      {
+        "certificate": "NuGet",
+        "strongName": null,
+        "values": [
+          "artifacts\\Microsoft.FSharp.Compiler*.nupkg"
+        ]
+      }
+    ],
+    "exclude": [
+        "artifacts\\*symbols.nupkg",
```

Comment:
Let's sign the symbols packages, too, in case we decide to start publishing those.